jest.config.js

```diff
@@ -2,10 +2,9 @@ module.exports = {
   bail: true,
   clearMocks: true,
   testTimeout: 30000,
-  testEnvironment: './tests/test-environment',
-  transformIgnorePatterns: [
-    '/node_modules/(?!@polkadot|@babel|@digitalbazaar)',
-  ],
+  testEnvironment: "./tests/test-environment",
+  transformIgnorePatterns: ["/node_modules/(?!@polkadot|@babel|multiformats)"],
+  workerIdleMemoryLimit: "1G",
```

Comment:
We run tests one by one (`inband`) in CI so is there some test taking upto 1G?

Reply:
Before this change was introduced, tests were crashing with `JavaScript heap out of memory` exception: https://github.com/docknetwork/sdk/actions/runs/8192487932/job/22404099356.